VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/models/linked-participant-panel-model.spec.ts

```diff
@@ -161,3 +162,32 @@ describe('LinkedParticipantPanelModel', () => {
         model = LinkedParticipantPanelModel.fromListOfPanelModels(pats, roomLabel, roomId);
     }
 });
+
+describe('LinkedParticipantPanelModel', () => {
```

Comment:
Might want to give the describe here a different value to make it clear the difference between the two fixtures